src/main.cpp

```diff
@@ -192,7 +192,14 @@ void main(void)
 	uint32_t *reset_vector;
 	reset_vector = (uint32_t *)RESET_VECTOR;
 	start_arc(*reset_vector);
```

Comment:
@bigdinotech Would it make a difference if the ARC is started after all the tasks?